src/components/Timeline.vue

```diff
@@ -300,4 +302,26 @@ export default defineComponent({
 .vue-slider-dot-tooltip-inner-top::after {
     display:none;
 }
+
+.vue-slider-dot-tooltip {
+    top: calc(100% - var(--slider-dot-size) / 3);
+}
+
+.vue-slider-dot-tooltip-inner {
+    top: calc(0px - var(--tooltip-line-height));
+    font-size: var(--sz-400);
+    transform: translateY(-100%);
+}
+
+.vue-slider-dot {
```

Comment:
I think you can avoid the important by setting the dotStyle variable

Reply:
J'ai essayé pendant un bout, mais malheureusement le `dotStyle` est mis sur le child en dessous d'où le style à bras est setté.

En d'autres termes, je ne pense pas qu'on peut contrôler le `style` qui est mis ici:
https://github.com/NightCatSama/vue-slider-component/blob/v4.0.0-beta.9/lib/vue-slider.vue#L78-L84
et `dot-style` affecte le child component en dessous de celui qu'on veut changer:
https://github.com/NightCatSama/vue-slider-component/blob/v4.0.0-beta.9/lib/vue-slider-dot.vue#L10

Peut-être que c'est moi qui ne réalise pas dequoi, mais pour l'instant je vais submit as-is, svp fix le si tu trouves comment. :)